src/common/src/array/data_chunk_iter.rs

```diff
@@ -249,49 +249,16 @@ impl Row {
         &EMPTY_ROW
     }
 
-    /// Serialize the row into a memcomparable bytes.
+    /// Serialize the row into value encoding bytes.
+    /// WARNING: If you want to serialize to a memcomparable format, use [`OrderedRow`]
     ///
     /// All values are nullable. Each value will have 1 extra byte to indicate whether it is null.
-    pub fn serialize(&self) -> Result<Vec<u8>, memcomparable::Error> {
-        let mut serializer = memcomparable::Serializer::new(vec![]);
-        for v in &self.0 {
-            serialize_datum_into(v, &mut serializer)?;
-        }
-        Ok(serializer.into_inner())
-    }
-
-    /// Serialize the row into a memcomparable bytes. All values must not be null.
-    pub fn serialize_not_null(&self) -> Result<Vec<u8>, memcomparable::Error> {
```

Comment:
Is it not used any more?

Reply:
yes. It is used for serializing a row that contains no null. I can't see it will be used in the future.